qt/aqt/fields.py

```diff
@@ -234,6 +235,10 @@ def saveField(self) -> None:
         if fld["rtl"] != rtl:
             fld["rtl"] = rtl
             self.change_tracker.mark_basic()
+        desc = f.fieldDesc.text()
+        if not "description" in fld or fld["description"] != desc:
```

Comment:
These references would benefit from a .get() as well

Reply:
Indeed. Thanks for that suggestion!